plugwise_usb/nodes/circle.py

```diff
@@ -834,7 +840,11 @@ async def node_info_update(
                 self.mac
             )
         if self._last_log_address != node_info.last_logaddress:
+            self._previous_log_address = self._last_log_address
+            _LOGGER.debug('%s update last_log_address %d -> %d', self.mac, 
+                          self._last_log_address, node_info.last_logaddress)            
             self._last_log_address = node_info.last_logaddress
```

Comment:
I discovered yesterday the log address at the `NodeInfoRespone` message is actually the pointer to the current log period which is not populated with the counters yet.
See also [this commit in the async branch](https://github.com/plugwise/python-plugwise-usb/commit/f6cc26d1104f072dfef298360470f26712fe1763)